### PR TITLE
Replaces custom objective for gorlex maurauderuresu

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

 their objective to emag the shuttle console was removed so this replaces IT. u GOTTA insert a DISK into the comms console THEN INSERT it into A PHONE then goddamn 5 MISSILES are launched at the station mad insanity. the missiles have crippling speed and r very slow so that even the MF with the worst reflexes can evade the bomb radius  in time

https://user-images.githubusercontent.com/95004236/184190852-765be108-2906-4b50-a47f-3487249fc32d.mp4

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: fulp_modules/features/antagonists/infiltrators/infil_items.dm [hunk 1]
```diff
@@ -238,9 +238,88 @@
...
+	if(!istype(emag_card, /obj/item/card/emag/silicon_hack))
+		return
+	if(!opened)
+		return
+	var/obj/item/card/emag/silicon_hack/hack_card = emag_card
+	hack_card.use_charge(user)
+	playsound(src, 'sound/machines/beep.ogg', 50, FALSE)
+
+/obj/item/missile_disk
+	name = "Missile Disk"
```
Comment (SgtHunk) [inline fulp_modules/features/antagonists/infiltrators/infil_items.dm:251]: lowercase

File: fulp_modules/features/antagonists/infiltrators/infil_items.dm [hunk 2]
```diff
@@ -238,9 +238,88 @@
...
+	item_flags = NOBLUDGEON
+	///Does the disk have the station coordinates?
+	var/stored = FALSE
+
+/obj/item/missile_disk/afterattack(atom/movable/victim, mob/living/carbon/human/user, proximity)
+	. = ..()
+	if(!proximity)
+		return
+	if(!istype(victim, /obj/machinery/computer/communications))
+		return
```
Comment (SgtHunk) [inline fulp_modules/features/antagonists/infiltrators/infil_items.dm:265]: this doesn't check if stored is TRUE

File: fulp_modules/features/antagonists/infiltrators/infil_items.dm [hunk 3]
```diff
@@ -238,9 +238,88 @@
...
+	to_chat(user, span_warning("Downloading station coordinates..."))
+	if(!do_after(user, 8 SECONDS))
+		return
+	playsound(src, 'sound/machines/beep.ogg', 50, FALSE)
+	to_chat(user, span_warning("Station coordinates successfully downloaded!"))
+	stored = TRUE
+
+/obj/item/missilephone
+	name = "Large Handphone"
+	desc = "Hello? Is this the police?"
```
Comment (SgtHunk) [inline fulp_modules/features/antagonists/infiltrators/infil_items.dm:276]: lowercase + dunno if the desc fits too well

**Review by SgtHunk — CHANGES_REQUESTED** [review]
i thought u needed 5 kills to unlock this

File: fulp_modules/features/antagonists/infiltrators/infil_items.dm (hunk 3 above)
Comment (SmoSmoSmoSmok) [reply fulp_modules/features/antagonists/infiltrators/infil_items.dm:276]: KUNGGGGGGGGGGGGGG FURY REFERENCE

Comment (Pepsilawn) [reply fulp_modules/features/antagonists/infiltrators/infil_items.dm]: > lowercase + dunno if the desc fits too well

it do

**Review by SgtHunk — CHANGES_REQUESTED** [review]

**Review by SgtHunk — APPROVED** [review]
ty

**Pull request merged**